larq_compute_engine/mlir/transforms/quantize_patterns.td

```diff
@@ -33,3 +32,66 @@ def : Pat<(LQ_Bconv2dOp
               $padding,
               $stride_height,
               $stride_width)>;
+
+// Checks if the value has only one user.
+def HasOneUse : Constraint<CPred<"$0.hasOneUse()">>;
+
+def : Pat<(LQ_Bconv2dOp
+              (TFL_DequantizeOp $input),
+              $filter,
+              $post_activation_multiplier,
+              $post_activation_bias,
+              $output_threshold,
+              $channels_in,
+              $dilation_height_factor,
+              $dilation_width_factor,
+              $fused_activation_function,
+              $pad_values,
+              $padding,
+              $stride_height,
+              $stride_width),
+          (LQ_Bconv2dOp $input,
+              $filter,
+              $post_activation_multiplier,
+              $post_activation_bias,
+              $output_threshold,
+              $channels_in,
+              $dilation_height_factor,
+              $dilation_width_factor,
+              $fused_activation_function,
+              $pad_values,
+              $padding,
+              $stride_height,
+              $stride_width),
+          [(HasOneUse $input)]>;
+
+def : Pat<(TFL_QuantizeOp
```

Comment:
We could merge this pattern into the one handling `post_activation_multiplier` and `post_activation_bias` but I opted to keep them separate to make sure that `post_activation_multiplier` and `post_activation_bias` will always converted correctly even if this pattern fails to match.